### ISSUE TITLE
Big loading time and freezes in Library after updating to 1.7.1

### ISSUE THREAD

**Issue by AlexElLeon1** [Jays2Kings/tachiyomiJ2K#1628]
### Steps to reproduce

1. Updated to 1.7.1
2. Open Library, takes between 20sec-3min to load.
3. Closing and opening the app causes Step 1 to repeat
4. Selecting a manga from the library causes the app to freeze for more than 20 sec. Manga chapter does not open and it's still on Library page
5. Exploring on extension's result in mangas loading only after the Library has loaded.
6. Going back to Library causes Step 1 to repeat

### Expected behavior

Previous to updating there were no issues with loading times (less than a second). Opening manga chapters was almost instant, never caused the app to freeze

### Actual behavior

Constant freezes and loading times.
The app does not crash, so crash registry shows nothing (see details for more explanations).

### Crash logs

App version: 1.7.0 (standard, cf432473e7, 106, 2023-09-28T00:08Z)
Android version: 11 (SDK 30)
Android build ID: RKQ1.201105.002
Device brand: lge
Device manufacturer: LGE
Device name: caymanlm
Device model: LM-G900N
Device product name: caymanlm

--------- beginning of system
--------- beginning of main
10-04 22:46:49.858  6652  6652 E e.tachiyomi.j2: Not starting debugger since process cannot load the jdwp agent.
10-04 22:46:50.212  6652  6708 E chromium: [1004/224650.212829:ERROR:variations_seed_loader.cc(69)] Failed to open file for reading. Errno: 2
10-04 22:46:50.681  6652  6652 E SQLiteLog: (283) recovered 1 frames from WAL file /data/data/eu.kanade.tachiyomi.j2k/databases/tachiyomi.db-wal
10-04 22:46:50.697  6652  6782 E SQLiteLog: (284) automatic index on C(manga_id)
10-04 22:46:50.697  6652  6782 E SQLiteLog: (284) automatic index on R(manga_id)
10-04 22:46:50.697  6652  6782 E SQLiteLog: (284) automatic index on B(manga_id)

### TachiyomiJ2K version

1.7.0

### Android version

Android 11

### Device

LG VELVET (LG G9)

### Other details

I saw posted Issue #1619 and tried the solutions posted. Downloaded update Beta v1.7.2-b01 and the issue persisted. 
Downgraded to 1.7.1 after doing a deep clean of the app and issue persisted, crash logs did not show anything more than:
--------- beginning of system 
--------- beginning of main.
Did another deep clean and installed 1.7.0 and issue persists. I will now try the suggested solution of going back to 1.6.6

I do not remember what version I had installed previous to updating, but I always downloaded new versions as they appeared so I assume it was 1.7.1

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] If this is an issue with an extension, I should be opening an issue in the [extensions repository](https://github.com/tachiyomiorg/tachiyomi-extensions/issues/new/choose).
- [X] I am reporting an issue exclusive to this fork. I have also checked that is not an issue on the [main version of Tachiyomi](https://github.com/tachiyomiorg/tachiyomi)
- [X] I have tried the [troubleshooting guide](https://tachiyomi.org/help/guides/troubleshooting/).
- [X] I have updated the app to version **[1.7.1](https://github.com/Jays2Kings/tachiyomiJ2K/releases/latest)**.
- [X] I have updated all installed extensions.
- [ ] I have admitted that I am a clown by having checked this box, as I have not read these acknowledgements.
- [X] I have filled out all of the requested information in this form.

**Comment by Jays2Kings** [comment]
As you saw the other issue is still open so you're just making a duplicate issue

**Comment by AlexElLeon1** [comment]
Ok
